### PR TITLE
Fix breaking viewHelper while changing camera

### LINKED ISSUE #463: Changing the camera projection type breaks the view helper
Changing the camera projection seems to break the view helper

https://github.com/user-attachments/assets/02288e03-90f3-41c3-9bc2-83af7f401105

### PR DESCRIPTION
**Pull request by arjxn-py**
Fixes #463 

https://github.com/user-attachments/assets/443497ee-5ce9-4731-ae5a-4f62bee59d0e

### CODE REVIEW

File: packages/base/src/3dview/mainview.tsx [hunk 1]
```diff
@@ -362,29 +362,33 @@ export class MainView extends React.Component<IProps, IStates> {
...
-      this._viewHelper = new ViewHelper(
-        this._camera,
-        this._renderer.domElement
-      );
-      this._viewHelper.center = this._controls.target;
+  private _createViewHelper() {
+    const viewHelperDiv = document.getElementById('viewHelper');
+    if (viewHelperDiv) {
+      viewHelperDiv.remove();
+    }
```
Comment (trungleduc) [inline packages/base/src/3dview/mainview.tsx:373]: This will remove the view helper of all opening documents. This is the expected behavior?

Comment (martinRenou) [reply packages/base/src/3dview/mainview.tsx:373]: Yeah, we can probably find a solution here that does not require using an `id` whatsoever, just `this.something.remove()`, something being the view helper container kept as a class property?

Comment (arjxn-py) [reply packages/base/src/3dview/mainview.tsx]: ThankYou both, I agree this was not the expected behaviour even if it was not noticeable while testing for multiple documents open.

I also realised that it was not necessary to remove viewHelperDiv and just updating it with `updateCamera` works as well.


Done in [`39f6648`](https://github.com/jupytercad/JupyterCAD/pull/469/commits/39f664819e084b58636c54d0a203ad4570cf3820)

File: packages/base/src/3dview/mainview.tsx [hunk 2]
```diff
@@ -362,29 +362,29 @@ export class MainView extends React.Component<IProps, IStates> {
...
 
-      this.divRef.current.appendChild(viewHelperDiv);
+    this._viewHelperDiv = viewHelperDiv;
 
-      viewHelperDiv.addEventListener('pointerup', event =>
-        this._viewHelper.handleClick(event)
-      );
-    }
-  };
+    this.divRef.current?.appendChild(this._viewHelperDiv);
```
Comment (martinRenou) [inline packages/base/src/3dview/mainview.tsx:382]: Just a question, how are we removing the old one now?

Comment (arjxn-py) [reply packages/base/src/3dview/mainview.tsx:382]: We're not removing it now as it was not necessary, just updating it with `updateCamera` works.

Comment (martinRenou) [reply packages/base/src/3dview/mainview.tsx:382]: What I mean is that you re-create a new `this._viewHelperDiv` in `updateCamera`, that you append to `this.divRef.current`.

Don't we need to remove the old `this._viewHelperDiv` that was created before?

Comment (arjxn-py) [reply packages/base/src/3dview/mainview.tsx:382]: image

Yes you're right, I just verified. Sorry that I overlooked, I'll be omw to fix it.

Comment (arjxn-py) [reply packages/base/src/3dview/mainview.tsx:391]: Done in [`4602e00`](https://github.com/jupytercad/JupyterCAD/pull/469/commits/4602e00cff6d62640a924ed800558c256c3c22fe)

Thank You for pointing this out :)

**Review by martinRenou — APPROVED** [review]
Thanks ! 🚀 

Failed UI tests are not related https://github.com/jupytercad/JupyterCAD/issues/471

**Pull request merged**